src/vma/netlink/netlink_wrapper.cpp

```diff
@@ -122,14 +120,15 @@ void netlink_wrapper::route_cache_callback(nl_object* obj)
 {
 	nl_logfunc( "---> route_cache_callback");
 	struct rtnl_route* route = (struct rtnl_route*) obj;
-	route_nl_event new_event(g_nl_rcv_arg.msghdr, route,g_nl_rcv_arg.netlink);
-	// notify only route events of main table
-	const netlink_route_info* info =  new_event.get_route_info();
-	if (info->table == ROUTE_MAIN_TABLE_ID) {
-		netlink_wrapper::notify_observers(&new_event, nlgrpROUTE);
-	}
-	else {
-		nl_logfunc("ROUTE events from non-main route table are filtered: table_id=%d", info->table);
+	if (route) {
+		int table_id = rtnl_route_get_table(route);
+		if ((table_id > 0) && (table_id != RT_TABLE_LOCAL)) {
```

Comment:
Please use RT_TABLE_UNSPEC instead of 0. 
=> done
    What about RT_TABLE_DEFAULT and table_id > RT_TABLE_LOCAL? You don't prevent these here.
=> Those are suppose to be supported by VMA.